helper/provutil/bastion.go

```diff
@@ -90,57 +91,61 @@ func getInstanceBastionHostFromEndpoint(ctx context.Context, deploymentID, nodeN
 	return b, nil
 }
 
-func getInstanceBastionHostFromRelationship(ctx context.Context, deploymentID, nodeName, depIdx string) (*sshutil.BastionHostConfig, error) {
-	bastionNodeName, err := deployments.GetTargetNodeForRequirement(ctx, deploymentID, nodeName, depIdx)
+func getInstanceBastionHostFromRelationship(ctx context.Context, deploymentID, nodeName, reqIdx string) (b *sshutil.BastionHostConfig, err error) {
+	bastNode, err := deployments.GetTargetNodeForRequirement(ctx, deploymentID, nodeName, reqIdx)
 	if err != nil {
-		return nil, err
+		return
```

Comment:
seems weird not to return error here and below ?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

Reply:
Hmm, makes sense, I've removed them.